ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
```

Comment:
To chime in un-invited, I think that it is safe to assume that this file contains proper yaml, and according to the [spec](http://www.yaml.org/spec/1.2/spec.html) true and false is encoded as the lower case literals 'true' and 'false (without quotes)